w1r3/java/src/main/java/W1R3.java

```diff
@@ -176,12 +177,22 @@ private void worker(
     var random = new Random(seed);
     var tracer = otelSdk.getTracer(SCOPE_NAME, SCOPE_VERSION);
     var meter = otelSdk.getMeter(SCOPE_NAME);
-    var histogram =
+    var bucketBoundaries = makeLatencyBoundaries();
+    var latencyHistogram =
         meter
             .histogramBuilder("ssb/w1r3/latency")
-            .setExplicitBucketBoundariesAdvice(makeLatencyBoundaries())
+            .setExplicitBucketBoundariesAdvice(bucketBoundaries)
             .setUnit("s")
             .build();
+
+    var cpuPerByteHistogram =
+        meter
+            .histogramBuilder("ssb/w1r3/cpu")
+            .setExplicitBucketBoundariesAdvice(bucketBoundaries)
```

Comment:
You are going to need different bucket boundaries for the CPU histogram.  Consider:

https://github.com/googleapis/storage-shared-benchmarking/blob/3a16030ea2b608bc022c71e0baa497d56bf65397/w1r3/go/main.go#L609-L622

Reply:
I'll change it, probably want to update the design doc as well. The doc says we can use the same buckets for latency and cpu.